include/persistent_data.hpp

```diff
@@ -217,6 +218,85 @@ class ConfigFile
         }
     }
 
+    void writeCurrentSessionData()
+    {
+        std::ofstream persistentFile(dumpFilename);
+        std::filesystem::perms permission =
+            std::filesystem::perms::owner_read |
+            std::filesystem::perms::owner_write |
+            std::filesystem::perms::group_read;
+        std::filesystem::permissions(filename, permission);
```

Comment:
This is wrong, Let's include https://github.com/ibm-openbmc/bmcweb/pull/1159/commits/b6453e856aec25b093e21174b20267e7aa00f592 here